_posts/0000-01-02-zehrayelkenci.md

```diff
@@ -0,0 +1 @@
+pr changes
```

Comment:
hello gitub
i know it is simple
but it is really confused
please accept it
lets say to me